app/models/user.rb

```diff
@@ -4,9 +4,11 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
   has_one_attached :csv_file
-
-  validates_uniqueness_of :email, :allow_blank => true, :case_sensitive => false
-  validates_uniqueness_of :phone_number, :allow_blank => true, :case_sensitive => false
+  validates_presence_of :first_name
+  validates_presence_of :last_name
+  validates_confirmation_of :password
```

Comment:
This should confirm presence of password too, no? 

Maybe we also need to do like:

`validate :password, presence: true`